release_images.yml

```diff
@@ -47,6 +47,16 @@ release_images:
                             # to images being published.
       force_release: False  # [Default: False] Set to True to force images to be published even if the same image
                             # has already been published. Re-released image will have minor version incremented by 1.
+    inference:
+      device_types: ["cpu", "gpu"]
+      python_versions: ["py36"]
+      os_version: "ubuntu16.04"
+      cuda_version: "cu101"
```

Comment:
Let's release with one cuda version at a time. 
1. Trigger release for `cu110` as the configuration on the master branch is up to date. 
2. Make changes to this pr for `cu102` and release again.